packages/instantsearch-ui-components/src/components/recommend-inner/DefaultHeader.tsx

```diff
@@ -0,0 +1,26 @@
+/** @jsx createElement */
+import { cx } from '../../lib';
+
+import type { InnerComponentProps, Renderer } from '../../types';
+
+export function createDefaultHeaderComponent({ createElement }: Renderer) {
+  return function DefaultHeader<TObject>(
```

Comment:
In InstantSearch we usually refer to this generic type as `THit`. I'd recommend renaming all `TObject` to `THit` for consistency.

Reply:
I'll check if those are equivalent, my understanding is that `TObject` is the object without `objectID` and `_score`